app/src/main/res/xml/preferences.xml

```diff
@@ -42,12 +42,13 @@
             android:summary="@string/aliases_desc"
             android:title="@string/aliases_name">
         </CheckBoxPreference>
-        <CheckBoxPreference
-            android:defaultValue="true"
-            android:key="enable-search"
-            android:summary="@string/search_desc"
-            android:title="@string/search_name">
-        </CheckBoxPreference>
+        <ListPreference
+            android:title="@string/search_engines"
+            android:summary="@string/choose_search_engine"
+            android:key="searchEngine"
```

Comment:
android-studio was refusing to compile it with a dash. I'll try it again and get back to you.